src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.Vector;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
 public class Duke {
+    public static final String LONG_LINE = "------------------------------------------------------------";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Initialize a vector to store all the tasks
+        Vector<Task> tasks = new Vector<>();
+
+        greeting();
+        printlnWithIndent(LONG_LINE);
+
+        Scanner in = new Scanner(System.in);
+        Boolean exit = false;
```

Comment:
maybe this boolean variables can be named to sound more like a boolean?

Reply:
Thanks for this advice. Will change in a new commit.